app/assets/stylesheets/application.scss

```diff
@@ -13,4 +13,21 @@
  *= require_tree .
  *= require_self
  */
- @import "bootstrap";
+//Importing bootstrap
+@import "bootstrap";
+/* Base */
+:root {
```

Comment:
There seems to be a typo in the variable name. It should be '--header-color' instead of '--head-color'.